genome_kit/data_manager.py

```diff
@@ -172,22 +167,23 @@ def FileIO(filename, desc, mode, size, quiet):
             )
             yield decorated
 
-class DefaultDataManager(DataManager):
-    """A minimal data manager implementation that retrieves files from the GCS samples bucket.
-    Uploads are not supported"""
-    def __init__(self, data_dir: str):
+class GCSDataManager(DataManager):
```

Comment:
should we just use boto for GCS as well?

Reply:
I guess we could. Hopefully the API is the same?

Reply:
I looked into it and I don't think it's worth the hassle. For example, the google client figures out where your credentials are automatically. We'd have to reimplement that in order to find the key+secret to pass to the client initializer.